### PR TITLE
glib{,-networking} bump to 2.58.0

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: srcpkgs/glib-networking/template [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+hostmakedepends="pkg-config"
 makedepends="ca-certificates gnutls-devel gsettings-desktop-schemas-devel
  libglib-devel libproxy-devel"
-depends="ca-certificates gsettings-desktop-schemas"
+depends="gsettings-desktop-schemas"
+checkdepends="glib"
 short_desc="Network-related giomodules for glib"
 maintainer="Enno Boland <gottox@voidlinux.eu>"
 license="LGPL-2.1-or-later"
 homepage="http://www.gnome.org/"
```
Comment (maxice8) [inline srcpkgs/glib-networking/template]: i think https://gitlab.gnome.org/GNOME/glib-networking/ is more suited.

**Comment by maxice8** [comment]
Seems like cross is failing

**Comment by Cogitri** [comment]
I suppose disabling gtk-doc for cross builds would fix it

**Comment by maxice8** [comment]
@pullmoll ok if we skip gtk-doc on cross arches ?

**Comment by pullmoll** [comment]
@maxice8 disabling gtk-doc is ok as very few people will need it anyway.

**Comment by maxice8** [comment]
On Fri, Sep 07, 2018 at 04:47:41AM -0700, Jürgen Buchmüller wrote:
> @maxice8 disabling gtk-doc is ok as very few people will need it anyway.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2363#issuecomment-419415027

ack!

**Comment by maxice8** [comment]
I took the liberty of fixing the $CROSS_BUILD conditional and pushing it

**Pull request closed without merging**